### ISSUE TITLE
Modal Images?

### ISSUE THREAD

**Issue by ignacio82** [Vimux/Mainroad#35]
Is it possible to [modal images](https://www.w3schools.com/howto/howto_css_modal_images.asp) to a blog post?

**Comment by Vimux** [comment]
Yes, it's possible, but Mainroad theme doesn't support modal images. So if you need this, you'll have to add this feature.
